### PR TITLE
A dataset that is compatible with torchvision

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

File: python/rikai/torch/transforms.py [hunk 1]
```diff
@@ -12,17 +12,18 @@
...
 
-from rikai.mixin import ToNumpy
+from rikai.mixin import ToNumpy, ToPIL
+
+__all__ = ["RikaiToTensor"]
 
 
-def convert_tensor(row):
-    """Convert row into rikai semantic objects."""
+def convert_tensor(row, use_pil: bool = False):
```
Comment (changhiskhan) [inline python/rikai/torch/transforms.py:25]: nit: the extra boolean parameter is a bit confusing because it took some tracing to figure out where it was being set to True etc.

**Review by changhiskhan — APPROVED** [review]
a few nits. no blockers

File: python/rikai/torch/data.py [hunk 2]
```diff
@@ -21,14 +21,12 @@
...
 # Third Party
-import numpy as np
 import torch
 from torch.utils.data import IterableDataset
 
 # Rikai
 import rikai.parquet
-from rikai.mixin import ToNumpy
-from rikai.torch.utils import convert_tensor
+from rikai.torch.transforms import RikaiToTensor, convert_tensor
```
[the remark this answers is not in the document]
Comment (eddyxu) [reply python/rikai/torch/data.py:29]: This is still used in the Rikai-made `DataLoader`, we can remove it once we start to use torch 1.8's iteratableDataset shuffler

File: python/rikai/torch/transforms.py (hunk 1 above)
Comment (eddyxu) [reply python/rikai/torch/transforms.py:25]: What would be an appropriate way to write parameter here? `use_pil = None` or making it a required parameter?
